microsoft-dynamics-365-business-central/1.0/modules/ROOT/pages/microsoft-dynamics-365-business-central-connector-examples.adoc

```diff
@@ -0,0 +1,550 @@
+| Callback Path | Path of the access token callback endpoint
+| Authorize Path | Path of the local HTTP endpoint that triggers the OAuth dance
+|===
+
+. Click *OK*.
+
+[[configure-global]]
+=== Configure a Global Element for the Properties File
+
+Configure a global element for the `mule-app.properties` file so that Mule knows
+where to find it:
+
+. Click the *Global Elements* tab and click *Create*.
+. In the *Choose Global Type* dialog, select *Configuration properties* and click *OK*.
+. In the *File* field, enter `mule.app.properties`.
+. Click *OK*.
+
+
+[[list-items]]
+== List Items
+
+This Mule flow lists all items.
+
+The first flow uses the following operations:
+
+* *HTTP Listener* +
+Accepts data from HTTP requests
+* *Parse Template* +
+Processes a template
+
+The second flow uses the following operations:
+
+* *HTTP Listener* +
+Accepts data from HTTP requests
+* *List Entities* +
+Retrieves a list of entities
+* *Set Payload* +
+Defines how Mule sets the payload
+----
+output application/json
+---
+payload.payload map (item) -> {
+		"id": item.id,
+		"name": item.displayName
+	}
+----
+
+
+image::list-items.png[Studio Flow for the List Entities Operation]
+
+=== XML for This Example
+
+Paste this code into the Studio XML editor to quickly load the flow for this example into your Mule app:
+
+[source,xml,linenums]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns:dynamics365-bc="http://www.mulesoft.org/schema/mule/dynamics365-bc" xmlns:http="http://www.mulesoft.org/schema/mule/http"
+      xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+http://www.mulesoft.org/schema/mule/dynamics365-bc http://www.mulesoft.org/schema/mule/dynamics365-bc/current/mule-dynamics365-bc.xsd">
+    <configuration-properties file="mule-app.properties"/>
+    <configuration-properties file="advanced.properties"/>
+
+    <http:listener-config name="HTTP_Listener_config" doc:name="HTTP Listener config" basePath="/">
+        <http:listener-connection host="0.0.0.0" port="8081" />
+    </http:listener-config>
+
+    <dynamics365-bc:config name="Business_Central_BasicAuth" doc:name="Microsoft Dynamics 365 - Business Central Config" >
+        <dynamics365-bc:basic-connection baseUri="${config.baseUri}" username="${auth.username}" password="${auth.password}" />
+    </dynamics365-bc:config>
+
+    <flow name="Home" >
+        <http:listener doc:name="/" config-ref="HTTP_Listener_config" path="/" />
+        <parse-template doc:name="Parse Template" location="dynamics365-bc-demo-template.html" />
```

Comment:
the file `dynamics365-bc-demo-template.html` is similar story to `advanced.properties`. I'm not sure about this, but I think this file wont be accessible anywhere from the docs unless we either include it for download somehow or paste it's contents directly into the docs 🤔 in that case we could add a section directing the reader to create the file and copy it's contents from the docs above the `=== XML for This Example` section. Maybe something like `=== Prepare the HTML template` section, explaining that using the html will allow the user to test the application using a graphical interface...

another alternative might be removing the "Home" flow from every example and just directing the user to use something like `cUrl` or `Postman` to hit the endpoints. 🤔 That's a bit less "fancy" though.